src/router/index.js

```diff
@@ -12,18 +12,18 @@ import Questionnaire from '../views/Questionnaire.vue'
 import TimelineView from '../views/TimelineView.vue'
 
 export default new Router({
-  mode: 'history',
-  scrollBehavior: () => ({ y: 0 }),
-  routes: [
-    { path: '/author/:authorId/:authorName?', component: List },
-    { path: '/category/:title', component: List },
-    { path: '/post/:slug', component: Article },
-    { path: '/q/:questionnaireId', component: Questionnaire },
-    { path: '/search/:keyword', component: List },
-    { path: '/section/:title', component: List },
-    { path: '/timeline/:title', component: TimelineView },
-    { path: '/tag/:tagId/:tagName?', component: List },
-    { path: '/topic/:topicId', component: List },
-    { path: '/', component: Home }
-  ]
-})
+    mode: 'history',
+    scrollBehavior: () => ({ y: 0 }),
+    routes: [
+        { path: '/author/:authorId/:authorName?', component: List },
+        { path: '/category/:title', component: List },
+        { path: '/post/:slug', component: Article },
```

Comment:
we should use 'story' and we will need the /story/:slug/index.html to the same route